src/library/scala/collection/IndexedSeqView.scala

```diff
@@ -61,7 +61,7 @@ object IndexedSeqView {
       this
     }
 
-    override def slice(from: Int, until: Int): Iterator[A] = {
+    override def sliceIterator(from: Int, until: Int): Iterator[A] = {
```

Comment:
can't you just do `override protected def `sliceIterator`?

Reply:
Whoops, you're right, I'll be more careful next time!